adafruit_thermal_printer/thermal_printer.py

```diff
@@ -287,15 +287,17 @@ def reset(self):
         # ESC + 'D' + tab stop value list ending with null to terminate.
         self.send_command("\x1BD\x04\x08\x10\x14\x18\x1C\x00")
 
-    def print(self, text, end="\n"):
+    def print(self, text, *, end="\n", encoding="utf-8"):
```

Comment:
Let's leave `end` as an argument that can be given by position as to not break the existing API.  We can revisit this in a later PR where we explicitly break the API.